src/compiler/checker.ts

```diff
@@ -981,6 +981,10 @@ namespace ts {
             return getExternalModuleMember(<ImportDeclaration>node.parent.parent.parent, node);
         }
 
+        function getTargetOfGlobalModuleExportDeclaration(node: GlobalModuleExportDeclaration): Symbol {
+            return node.parent.symbol;

```

Comment:
Hrm, yes. You could have a module jquery.d.ts and then an augmentation of it in another file. I assume that has some impact here?